client/my-sites/comments/comment/comment-header.jsx

```diff
@@ -0,0 +1,50 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+import noop from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CommentAuthor from './comment-author';
+import CommentAuthorMoreInfo from './comment-author-more-info';
+import FormCheckbox from 'components/forms/form-checkbox';
+
+export const CommentHeader = ( {
+	commentId,
+	isBulkMode,
+	isEditMode,
+	isExpanded,
+	isSelected,
+	siteId,
+	toggleExpanded,
+} ) => (
+	<div className="comment__header" onClick={ ! isExpanded && ! isBulkMode ? toggleExpanded : noop }>
+		{ isBulkMode && (
+			<label className="comment__bulk-select">
+				<FormCheckbox checked={ isSelected } onChange={ noop } />
+			</label>
+		) }
+
+		<CommentAuthor { ...{ commentId, isExpanded, siteId } } />
```

Comment:
Will siteId ever not match the current selectedSiteId?

Reply:
If not, we don't need to pass this down the chain, and CommentAuthor can pull this itself.

Reply:
Great point, never thought of it!